project.md

```diff
@@ -11,41 +11,96 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+This need is domain-specific - it focuses on a subset of word documents that have an implicit or designed order and modules that can be identified by keywords. The limited expressiveness is also guaranteed because this task is focused on reordering existing data, rather than editing the data itself.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I interviewed my high school friend (who works in sales/marketing) about her experience with computers. She recently booked a flight for a trip and ran into a lot of issues with booking flights. Our conversation then verged into planning for travel, which is how this idea came about.
+
+This idea is exciting because it involves balancing between “programming” for users that are likely not programmers with enough notation to prevent confusion (like the Perl vs AppleScript example from the last homework). It might also get my friend to be less fearful of breaking computer-related things and try writing scripts to help her tasks without feeling like she is “programming”.
+
+Taking this idea to fruition seems to involve similar language design decisions to many text processing DSLs that currently exist. Namely, one domain that I am thinking of is choose your own adventure writers like Ink and ChoiceScript with the way their hierarchies and orderings are portrayed. As a newbie DSL implementer, this similarity allows me to reference examples to improve on my design. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Brainstorm, planning, and text processing
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+Writing in the DSL would optimatlly live in a GUI similar to ContextFree for end-users. This is because the vast amount of people in my domain that would not be familiar with programming. 
+
+In terms of syntax, it should also be relatively human-readable. The user should first identify the file for reordering in the program, just like how users would need to identify images for image processing DSLs. The documents I am focusing on are those intended to have some order. Additionally, these structures are repeatable and can be identifiable by keywords. For example, an outline should have paragraphs, and within paragraphs, there should be arguments, each supported by a quote and an analysis. Text that should be in the same structure should also be identifiable by certain keywords.
+
```

Comment:
I remember that you mentioned in class that you potentially wanted input on how to create syntax for your DSL. Based on this section, I feel that I have a general understanding of what you are envisioning but I don't think I understand it well enough to fully see it myself. Next studio in class, we should talk explain our projects in deeper detail to each other so that we can help each other out to the best of our ability